bundles/org.openhab.binding.carnet/src/main/java/org/openhab/binding/carnet/internal/handler/CarNetVehicleHandler.java

```diff
@@ -0,0 +1,643 @@
+            }
+
+            // Create services
+            services.clear();
+            addService(serviceAvailability.statusData, CNAPI_SERVICE_VEHICLE_STATUS_REPORT,
+                    new CarNetVehicleServiceStatus(this, api));
+            addService(serviceAvailability.carFinder, CNAPI_SERVICE_CAR_FINDER,
+                    new CarNetVehicleServiceCarFinder(this, api));
+            addService(serviceAvailability.rlu, CNAPI_SERVICE_REMOTE_LOCK_UNLOCK,
+                    new CarNetVehicleServiceRLU(this, api));
+            addService(serviceAvailability.charger, CNAPI_SERVICE_REMOTE_BATTERY_CHARGE,
+                    new CarNetVehicleServiceCharger(this, api));
+            addService(serviceAvailability.clima, CNAPI_SERVICE_REMOTE_PRETRIP_CLIMATISATION,
+                    new CarNetVehicleServiceClimater(this, api));
+            addService(serviceAvailability.tripData, CNAPI_SERVICE_REMOTE_TRIP_STATISTICS,
+                    new CarNetVehicleServiceTripData(this, api));
+            addService(serviceAvailability.destinations, CNAPI_SERVICE_MY_AUDI_DESTINATIONS,
+                    new CarNetVehicleServiceDestinations(this, api));
+
+            if (!channelsCreated) {
+                // Add additional channels
+                Map<String, ChannelIdMapEntry> channels = new LinkedHashMap<>();
+                for (Map.Entry<String, CarNetVehicleBaseService> s : services.entrySet()) {
+                    s.getValue().createChannels(channels);
+                }
+
+                logger.debug("{}: Creating {} channels", thingId, channels.size());
+                ArrayList<ChannelIdMapEntry> channelList = new ArrayList<>(channels.values());
+                /*
+                 * try (FileWriter myWriter = new FileWriter("carnetChannels.MD")) {
+                 * String lastGroup = "";
+                 * for (ChannelIdMapEntry e : channelList) {
+                 * String group = lastGroup.equals(e.groupName) ? "" : e.groupName;
+                 * String s = String.format("| %-12.12s | %-23.23s | %-20.20s | %-7s | %-85s |\n", group,
+                 * e.channelName, e.itemType, e.readOnly ? "yes" : "no", e.getDescription());
+                 * myWriter.write(s);
+                 * lastGroup = e.groupName;
+                 * }
+                 * } catch (IOException e) {
+                 * }
+                 */
+                createChannels(channelList);
+                channelsCreated = true;
+            }
+        } catch (CarNetException e) {
+            CarNetApiErrorDTO res = e.getApiResult().getApiError();
+            if (res.description.contains("disabled ")) {
+                // Status service in the vehicle is disabled
+                String message = "Status service is disabled, check data privacy settings in MMI: " + res;
+                logger.debug("{}: {}", thingId, message);
+                updateStatus(ThingStatus.OFFLINE, ThingStatusDetail.CONFIGURATION_PENDING, message);
```

Comment:
The log message could be removed, as the status update is already logged by the framework.

The state change originated by updateStatus() is logged to events.log. Including the status detail message.